tests/integration/ceph_base_deploy_test.go

```diff
@@ -74,6 +74,16 @@ func checkIfRookClusterIsHealthy(s suite.Suite, testClient *clients.TestClient,
 	require.Nil(s.T(), err)
 }
 
+func checkIfRookClusterHasHealthyIngress(s suite.Suite, k8sh *utils.K8sHelper, clusterNamespace string) {
+	logger.Infof("Testing ingress %s health", clusterNamespace)
+	status, err := k8sh.GetResourceStatus("Ingress", clusterNamespace + "-dashboard", clusterNamespace)
+	if err != nil {
+		logger.Infof("Failed to find expected ingress %s", err)
+		s.T().Fail()
```

Comment:
This assert can replace lines 80-83
```suggestion
		assert.NoError(s.T(), err)
```

Reply:
@travisn pushed that change